old-tutorials-in-development/tutorials-in-development/R/dc-spatio-temporal-intro/04-intro-CRS-projection.Rmd

```diff
@@ -190,18 +187,20 @@ units are Degrees and the coordinate system itself is **latitude** and
 the central meridian on the globe (0,0).
 
 
-```{r lat-long-example }
+```{r lat-long-example, fig.cap="A map of the world plotted with X and Y axes labeled in degrees."}
 
 library(rgdal)
 library(ggplot2)
 library(rgeos)
 library(raster)
 
-# be sure to set your working directory
-# setwd("~/Documents/data")
+# set working directory to ensure R can find the file we wish to import and where 
+# we want to save our files. Be sure to move the unzipped download into your working directory! 
+wd <- "~/Documents/data/" #This will depend on your local environment 
+setwd(wd) 
 
 # read shapefile
-worldBound <- readOGR(dsn="Global/Boundaries/ne_110m_land",
+worldBound <- readOGR(dsn="ne_110m_land",
```

Comment:
When I tried to run the code with the `paste0()`, I kept getting error messages. But it worked just fine without the `paste0()`... My wd always seems to have trouble with the `paste0` function so that could be why I got an error but I'm also wondering if `readOGR` just doesn't need it if the wd is already set.

Reply:
Ahh, ok. I do seem to remember readOGR having some filename issues when I have used it before. I will test this out on my end. Thanks for trying it out